viam_cartographer_test.go

```diff
@@ -285,7 +286,32 @@ func TestNewFeatureFlag(t *testing.T) {
 		test.That(t, svc.Close(context.Background()), test.ShouldBeNil)
 	})
 
-	t.Run("Failed creation of cartographer slam service with invalid sensor "+
+	t.Run("Successful creation of cartographer slam service with good lidar with feature flag enabled, with IMU", func(t *testing.T) {
```

Comment:
[nit] I dont think you need the `with feature flag enabled` as these tests are all grouped under the TestNewFeatureFlag